common/pom.xml

```diff
@@ -44,6 +44,11 @@
             <artifactId>commons-io</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
It looks like this dependency is not used, so it can be removed

Reply:
yes, i agree